### PR TITLE
New content for PHP in Spanish

### PR DESCRIPTION
**Pull request by BrunoGGM**
## What does this PR do?
Add resource

### Description
Learn PHP with practical examples

### Why is this valuable (or not)?
New content for PHP in Spanish

### How do we know it's really free?
It is available at openlibra.com under a Creative Commons Attribution-ShareAlike 2.5 Spain license.

### For book lists, is it a book? For course lists, is it a course? etc.
A book

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-es.md [hunk 1]
```diff
@@ -289,6 +289,7 @@
 
 * [Domain Driven Design with PHP (Diseño guiado por Dominio con PHP)](https://www.youtube.com/playlist?list=PLfgj7DYkKH3DjmXTOxIMs-5fcOgDg_Dd2) - Carlos Buenosvinos Zamora (YouTube playlist)
 * [Manual de estudio introductorio al lenguaje PHP procedural](https://web.archive.org/web/20140209203630/http://www.cursosdeprogramacionadistancia.com/static/pdf/material-sin-personalizar-php.pdf) - Eugenia Bahit (PDF)
+* [PHP a través de ejemplos](https://openlibra.com/es/book/programacion-en-php-a-traves-de-ejemplos) - Manuel Palomo Duarte, Ildefonso Montero Pérez (HTML) [(PDF)](https://openlibra.com/es/book/download/programacion-en-php-a-traves-de-ejemplos)
```
Comment (shrn01) [inline books/free-programming-books-es.md:292]:
```suggestion
* [Programación en PHP a través de ejemplos](https://openlibra.com/es/book/programacion-en-php-a-traves-de-ejemplos) - Manuel Palomo Duarte, Ildefonso Montero Pérez (HTML) [(PDF)](https://openlibra.com/es/book/download/programacion-en-php-a-traves-de-ejemplos)
```

Don't change the book name

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-es.md [hunk 2]
```diff
@@ -289,6 +289,7 @@
 
 * [Domain Driven Design with PHP (Diseño guiado por Dominio con PHP)](https://www.youtube.com/playlist?list=PLfgj7DYkKH3DjmXTOxIMs-5fcOgDg_Dd2) - Carlos Buenosvinos Zamora (YouTube playlist)
 * [Manual de estudio introductorio al lenguaje PHP procedural](https://web.archive.org/web/20140209203630/http://www.cursosdeprogramacionadistancia.com/static/pdf/material-sin-personalizar-php.pdf) - Eugenia Bahit (PDF)
+* [Programación en PHP a través de ejemplos](https://openlibra.com/es/book/programacion-en-php-a-traves-de-ejemplos) - Manuel Palomo Duarte, Ildefonso Montero Pérez (HTML) [(PDF)](https://openlibra.com/es/book/download/programacion-en-php-a-traves-de-ejemplos)
```
Comment (shrn01) [inline books/free-programming-books-es.md:292]: Change the ordering to make it alphabetically ordered

**Review by shrn01 — APPROVED** [review]

File: books/free-programming-books-es.md [hunk 3]
```diff
@@ -291,6 +291,7 @@
 * [Manual de estudio introductorio al lenguaje PHP procedural](https://web.archive.org/web/20140209203630/http://www.cursosdeprogramacionadistancia.com/static/pdf/material-sin-personalizar-php.pdf) - Eugenia Bahit (PDF)
 * [PHP y Programación orientada a objetos](https://styde.net/php-y-programacion-orientada-a-objetos/) - Duilio Palacios (HTML)
 * [POO y MVC en PHP](https://bibliotecafacet.com.ar/wp-content/uploads/2014/12/eugeniabahitpooymvcenphp.pdf) - Eugenia Bahit (PDF)
+* [Programación en PHP a través de ejemplos](https://openlibra.com/es/book/programacion-en-php-a-traves-de-ejemplos) - Manuel Palomo Duarte, Ildefonso Montero Pérez (HTML) [(PDF)](https://openlibra.com/es/book/download/programacion-en-php-a-traves-de-ejemplos)
```
Comment (eshellman) [inline books/free-programming-books-es.md:294]: just one link please - these are both from the same site

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**